spot_ros2_control/hardware/src/spot_hardware_interface.cpp

```diff
@@ -373,6 +414,128 @@ void SpotHardware::stop_state_stream() {
   state_stream_started_ = false;
 }
 
+bool SpotHardware::start_command_stream() {
+  if (command_stream_started_) {
+    RCLCPP_INFO(rclcpp::get_logger("SpotHardware"), "Command stream has already been started!");
+    return true;
+  }
+  // Start command streaming
+  auto robot_command_stream_client_resp = robot_->EnsureServiceClient<::bosdyn::client::RobotCommandClient>();
+  if (!robot_command_stream_client_resp) {
+    RCLCPP_ERROR(rclcpp::get_logger("SpotHardware"), "Could not create robot command client");
+    return false;
+  }
+  command_client_ = robot_command_stream_client_resp.response;
+
+  auto endpoint_result = robot_->StartTimeSyncAndGetEndpoint();
+  if (!endpoint_result) {
+    RCLCPP_ERROR(rclcpp::get_logger("SpotHardware"), "Could not get timesync endpoint");
+    return false;
+  }
+
+  command_client_->AddTimeSyncEndpoint(endpoint_result.response);
+
+  RCLCPP_INFO(rclcpp::get_logger("SpotHardware"), "Robot Command Client successfully created!");
+
+  bosdyn::api::RobotCommand joint_command = ::bosdyn::client::JointCommand();
+  auto joint_res = command_client_->RobotCommand(joint_command);
+  if (!joint_res.status) {
+    RCLCPP_ERROR(rclcpp::get_logger("SpotHardware"), "Failed to activate joint control mode");
+    RCLCPP_ERROR(rclcpp::get_logger("SpotHardware"), "Message: %s", joint_res.status.DebugString().c_str());
+    return false;
+  }
+
+  auto robot_command_stream_resp = robot_->EnsureServiceClient<::bosdyn::client::RobotCommandStreamingClient>();
+  if (!robot_command_stream_resp) {
+    RCLCPP_ERROR(rclcpp::get_logger("SpotHardware"), "Could not create robot command streaming client");
+    return false;
+  }
+  command_stream_service_ = robot_command_stream_resp.response;
+
+  RCLCPP_INFO(rclcpp::get_logger("SpotHardware"), "Robot Command Streaming Client successfully created!");
+
+  // Fill in the parts of the joint streaming command request that are constant.
+  auto* joint_cmd = joint_request_.mutable_joint_command();
+  // Gain values https://github.com/boston-dynamics/spot-cpp-sdk/blob/master/cpp/examples/joint_control/constants.hpp
```

Comment:
@tcappellari-bdai should this comment live next to gain constant definition in `spot_joint_map.hpp`?